ddtrace/contrib/openai/_endpoint_hooks.py

```diff
@@ -203,6 +205,7 @@ class _CompletionHook(_BaseCompletionHook):
 
     def _record_request(self, pin, integration, span, args, kwargs):
         super()._record_request(pin, integration, span, args, kwargs)
+        span.span_type = SpanTypes.LLM
```

Comment:
This is being set manually here because OpenAI spans are created by the same patched method, but the underlying endpoints can be non-LLM (ex: embeddings/images/audio vs completion/chat)